packages/flutter/test/material/reorderable_list_test.dart

```diff
@@ -541,6 +545,138 @@ void main() {
+          home: SizedBox(
+            height: 100,
+            width: 100,
+            child: configuration,
+          ),
+        ));
+
+          final TestGesture gesture = await tester.startGesture(tester.getCenter(find.byType(ReorderableListView)));
+          await gesture.moveBy(_kGestureOffset);
+          // Move back to original position.
+          await gesture.moveBy(Offset(0, -_kGestureOffset.dy));
+          await tester.pump();
+          await tester.pump(const Duration(milliseconds: 500));
+
+          final TestGesture overscrollGesture = await tester.startGesture(tester.getCenter(find.byType(ReorderableListView)));
+          await overscrollGesture.moveBy(_kGestureOffset);
+          await tester.pump();
+          await tester.pump(const Duration(milliseconds: 500));
+
+          ScrollPhysics physics;
+          await tester.pumpWidget(
+            Builder(
+              builder: (BuildContext context) {
+                physics = configuration.behavior.getScrollPhysics(context);
+                return Container();
+              },
+            ),
+          );
+          expect(physics, isNotNull);
+        });
+
+        testWidgets('Default ReorderableListViews are not always scrollable', (WidgetTester tester) async {
+          final ScrollConfiguration configuration = ScrollConfiguration(
+           behavior: const ScrollBehavior(),
+           child: ReorderableListView(
+            scrollDirection: Axis.horizontal,
+            children: const <Widget>[],
+            onReorder: (int oldIndex, int newIndex) {},
+          ));
+
+          ScrollPhysics physics;
+          await tester.pumpWidget(
+            Builder(
+              builder: (BuildContext context) {
+                physics = configuration.behavior.getScrollPhysics(context);
+                return Container();
+              },
+            ),
+          );
+          expect(physics, isNotNull);
+        });
+
+        testWidgets('physics:AlwaysScrollableScrollPhysics overrides default behavior', (WidgetTester tester) async {
+          bool scrolled = false;
+          final ReorderableListView listViewDefault = ReorderableListView(
+            children: const <Widget>[],
+            onReorder: (int oldIndex, int newIndex) {},
+          );
+
+          final ReorderableListView listViewOverride = ReorderableListView(
+            physics: const AlwaysScrollableScrollPhysics(),
+            children: const <Widget>[],
+            onReorder: (int oldIndex, int newIndex) {},
+          );
+
+          await tester.pumpWidget(
+            MaterialApp(
+              home: Directionality(
+              textDirection: TextDirection.ltr,
+              child: NotificationListener<OverscrollNotification>(
+                onNotification: (OverscrollNotification message) { scrolled = true; return false; },
+                child: listViewDefault,
+              ),
+            ),
+            )
+          );
```

Comment:
indentations and trailing commas:
```suggestion
            MaterialApp(
              home: Directionality(
                textDirection: TextDirection.ltr,
                child: NotificationListener<OverscrollNotification>(
                  onNotification: (OverscrollNotification message) { scrolled = true; return false; },
                  child: listViewDefault,
                ),
              ),
            ),
          );
```